x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -169,13 +169,10 @@ const Unprivileged = ({ unprivilegedIndices }: { unprivilegedIndices: string[] }
   />
 );
 
-const EmptySecurityFindingsPrompt = ({
-  kspmIntegrationLink,
-  cspmIntegrationLink,
-}: {
-  kspmIntegrationLink?: string;
-  cspmIntegrationLink?: string;
-}) => {
+const EmptySecurityFindingsPrompt = () => {
+  const kspmIntegrationLink = useCspIntegrationLink(KSPM_POLICY_TEMPLATE);
+  const cspmIntegrationLink = useCspIntegrationLink(CSPM_POLICY_TEMPLATE);
```

Comment:
moved hooks to here for simplicity, also added `isDisabled` state on the links while queries are fetching.